pom.xml

```diff
@@ -202,6 +203,25 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>com.diffplug.spotless</groupId>
+                <artifactId>spotless-maven-plugin</artifactId>
+                <version>2.22.5</version>
+                <configuration>
+                    <java>
+                        <toggleOffOn>
+                            <off>@format:off</off>
+                            <on>@format:on</on>
+                        </toggleOffOn>
+                        <palantirJavaFormat>
+                            <version>2.24.0</version>
+                        </palantirJavaFormat>
+                        <licenseHeader>
+                            <file>${license-header.dir}/license-header</file>
+                        </licenseHeader>
+                    </java>
```

Comment:
you should add a https://github.com/diffplug/spotless/tree/main/plugin-maven#how-can-i-enforce-formatting-gradually-aka-ratchet so since otherwise now everting fails since the old ones are not reformatted.

Reply:
This is intentional. a full reformat of the project will be committed to develop directly. so for all branches from that point on this will not be a problem. all currently existing branches can rebase to the state after this PR and then do a full reformat themself or merge in develop

Reply:
did you check that the format passes?